colorama/initialise.py

```diff
@@ -53,6 +53,14 @@ def deinit():
         sys.stderr = orig_stderr
 
 
+@contextlib.contextmanager
+def colorama_text(*args):
```

Comment:
Maybe we should add **kwargs too to pass to 'init' so we can use the context manager with the keywords